bokeh/query.py

```diff
@@ -41,7 +41,7 @@ class LEQ(object): pass
 class NEQ(object): pass
 
 
-def match(obj, selector, context={}):
+def match(obj, selector, context=None):
```

Comment:
`context` may need to be passed explicitly to the recursive calls to `match` now